stash/test/api/collators.api.test.ts

```diff
@@ -41,16 +41,15 @@ function validateReturnedDate(date :string, format : string) {
 
 //perhaps those addresses may change when data is upgraded!
 
-const oldCollatorAddress = "5EtT1Psa48f9KSNMZqZgwPj6guNULVeRUzcA9w1Kk8DkP9iZ";
-const collatorAddress =  "5HNnu1WVhPCxdD8uHFoRyZ2qJE8hWQL3aaG8D1gaYhK3j8a7";
+const collatorAddress =  "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac";
```

Comment:
Added this address that was communicated with the team. Can (and probably should) be changed, left the //todo: in code.

Reply:
Checked with @goncer removed todo